app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1267,6 +1322,50 @@ protected void onResume() {
         checkEnterKeyPref();
     }
 
+
+    public void setReminder(String des , String time){
+
+        LinkedList<String> list = new LinkedList<>();
+        Matcher matcher = Pattern.compile("\\d+").matcher(time);
+        while (matcher.find()) {
+            list.add(matcher.group());
+        }
+
+        String[] timearray;
+
+        timearray = list.toArray(new String[list.size()]);
+        System.out.println(Arrays.toString(timearray));
```

Comment:
@mayank408: On most devices System.out.println gets redirected to LogCat and printed using Log.i(). But this may not be true on very old or custom Android versions. So please add LogCat itself.